### PR TITLE
Install the profiler in a version-safe manner

### PR DESCRIPTION
**Pull request by mjcarroll**
# Bug Report

Fixes issue #167
Fixes issue #82 

## Summary

This makes the profiler script get installed in a versioned subdirectory, which will enable side-by-side installs.

In this case, it follows along with our CLI executable strategy and installs to `install/lib/ignition/ignition-common4/ign_remotery_vis`

This additionally makes the profiler script relocatable by removing absolute paths from the CMake configuration process.

## Checklist
- [x] Signed all commits for DCO
- [x] ~Added tests~
- [x] Updated documentation (as needed)
- [x] ~Updated migration guide (as needed)~
- [x] `codecheck` passed (See
  [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See
  [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review
[another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+archived%3Afalse+)
to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: tutorials/profiler.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ From terminal 2, open the visualizer using one of the following commands
 
 ```{.sh}
 # Use a launcher script (Linux and macOS)
-ign_remotery_vis
+/usr/lib/ignition/ignition-common4/ign_remotery_vis
```
Comment (scpeters) [inline tutorials/profiler.md:89]: this path is incorrect on macOS; it installs to `/usr/local`

**Review by scpeters — APPROVED** [review]
looks fine except for the tutorial path is incorrect on macOS

File: tutorials/profiler.md (hunk 1 above)
Comment (chapulina) [reply tutorials/profiler.md]: Added instructions in 162832f79647bb04c6489b834472f20f69e09024

**Review by chapulina — APPROVED** [review]

File: tutorials/profiler.md (hunk 1 above)
Comment (scpeters) [reply tutorials/profiler.md]: also, the Ubuntu debian install path is `/usr/lib/x86_64-linux-gnu/ignition/ignition-common4/ign_remotery_vis`

Comment (scpeters) [reply tutorials/profiler.md]: well done

**Pull request merged**